+ source ./common/disk_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment!'
+ message='Start to prepare the test environment!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment!'
Fri Oct  6 21:04:19 2023 - INFO  - Start to prepare the test environment!
+ DNF_INSTALL lvm2
+ pkgs=lvm2
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs lvm2 --node 1 --tempfile ''
Fri Oct  6 21:04:19 2023 - INFO  - pkgs:(lvm2) is already installed
+ tmpfile2=
+ '[' -z '' ']'
+ tmpfile=
+ check_free_disk
++ TEST_DISK
++ id=1
++ python3 /root/mugen/libs/locallibs/get_test_device.py --device disk --node 1
+ disks='fd0 sr0'
+ disk_list=($disks)
+ local_disk=fd0
+ local_disk1=sr0
+ local_disk2=
+ local_disk3=
++ cat /etc/os-release
++ grep VERSION_ID
++ awk -F = '{print$NF}'
++ awk -F '"' '{print$2}'
+ version_id=23.09
+ LOG_INFO 'End to prepare the test environment!'
+ message='End to prepare the test environment!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment!'
Fri Oct  6 21:04:20 2023 - INFO  - End to prepare the test environment!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start executing testcase!'
+ message='Start executing testcase!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start executing testcase!'
Fri Oct  6 21:04:20 2023 - INFO  - Start executing testcase!
+ pvcreate -y /dev/fd0
  Device open 2:0 has no path names.
  Cannot use /dev/fd0: device not found
+ CHECK_RESULT 5
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_pvresize_001.sh line 31'
+ message='oe_test_lvm2_pvresize_001.sh line 31'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_pvresize_001.sh line 31'
Fri Oct  6 21:04:20 2023 - ERROR - oe_test_lvm2_pvresize_001.sh line 31
+ '[' 0 -eq 1 ']'
+ return 0
+ grep 'Physical volume "/dev/fd0" changed'
+ pvresize -y /dev/fd0 --setphysicalvolumesize 50MB
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_pvresize_001.sh line 33'
+ message='oe_test_lvm2_pvresize_001.sh line 33'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_pvresize_001.sh line 33'
Fri Oct  6 21:04:20 2023 - ERROR - oe_test_lvm2_pvresize_001.sh line 33
+ '[' 0 -eq 1 ']'
+ return 0
+ pvs
+ sed -n 3p
+ grep 50.00m
+ awk '{print$5}'
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_pvresize_001.sh line 35'
+ message='oe_test_lvm2_pvresize_001.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_pvresize_001.sh line 35'
Fri Oct  6 21:04:20 2023 - ERROR - oe_test_lvm2_pvresize_001.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ pvresize -y /dev/fd0 --reportformat basic
+ grep 'Physical volume "/dev/fd0" changed'
  Cannot use /dev/fd0: device type is unknown
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_pvresize_001.sh line 37'
+ message='oe_test_lvm2_pvresize_001.sh line 37'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_pvresize_001.sh line 37'
Fri Oct  6 21:04:20 2023 - ERROR - oe_test_lvm2_pvresize_001.sh line 37
+ '[' 0 -eq 1 ']'
+ return 0
+ pvresize -y /dev/fd0 --driverloaded y
+ grep 'Physical volume "/dev/fd0" changed'
  Cannot use /dev/fd0: device type is unknown
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_pvresize_001.sh line 39'
+ message='oe_test_lvm2_pvresize_001.sh line 39'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_pvresize_001.sh line 39'
Fri Oct  6 21:04:20 2023 - ERROR - oe_test_lvm2_pvresize_001.sh line 39
+ '[' 0 -eq 1 ']'
+ return 0
+ pvresize --version
+ grep 'LVM version'
  LVM version:     2.03.21(2) (2023-04-21)
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ pvresize --longhelp
+ grep 'Common variables for lvm'
  Common variables for lvm:
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ pvresize -y /dev/fd0 --nolocking
+ grep 'Physical volume "/dev/fd0" changed'
  Cannot use /dev/fd0: device type is unknown
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_pvresize_001.sh line 45'
+ message='oe_test_lvm2_pvresize_001.sh line 45'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_pvresize_001.sh line 45'
Fri Oct  6 21:04:20 2023 - ERROR - oe_test_lvm2_pvresize_001.sh line 45
+ '[' 0 -eq 1 ']'
+ return 0
+ pvresize -y /dev/fd0 -q
+ grep 'Physical volume "/dev/fd0" changed'
  Cannot use /dev/fd0: device type is unknown
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_pvresize_001.sh line 47'
+ message='oe_test_lvm2_pvresize_001.sh line 47'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_pvresize_001.sh line 47'
Fri Oct  6 21:04:20 2023 - ERROR - oe_test_lvm2_pvresize_001.sh line 47
+ '[' 0 -eq 1 ']'
+ return 0
+ pvresize -y /dev/fd0 --verbose
+ grep 'Physical volume "/dev/fd0" changed'
  Cannot use /dev/fd0: device type is unknown
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lvm2_pvresize_001.sh line 49'
+ message='oe_test_lvm2_pvresize_001.sh line 49'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_lvm2_pvresize_001.sh line 49'
Fri Oct  6 21:04:20 2023 - ERROR - oe_test_lvm2_pvresize_001.sh line 49
+ '[' 0 -eq 1 ']'
+ return 0
+ '[23.09' = '22.03]'
oe_test_lvm2_pvresize_001.sh: line 50: [23.09: command not found
+ LOG_INFO 'End executing testcase!'
+ message='End executing testcase!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End executing testcase!'
Fri Oct  6 21:04:20 2023 - INFO  - End executing testcase!
+ CASE_RESULT 0
+ case_re=0
+ test -z 8
+ test 8 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Oct  6 21:04:20 2023 - ERROR - failed to execute the case.
+ exit 8
+ post_test
+ LOG_INFO 'Start environment cleanup.'
+ message='Start environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environment cleanup.'
Fri Oct  6 21:04:20 2023 - INFO  - Start environment cleanup.
+ pvremove -f /dev/fd0
  Device open 2:0 has no path names.
  Cannot use /dev/fd0: device not found
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z '' ]]
+ [[ -z '' ]]
+ LOG_WARN 'no thing to do.'
+ message='no thing to do.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level warn --message 'no thing to do.'
Fri Oct  6 21:04:20 2023 - WARN  - no thing to do.
+ return 0
+ LOG_INFO 'Finish environment cleanup.'
+ message='Finish environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup.'
Fri Oct  6 21:04:20 2023 - INFO  - Finish environment cleanup.
